### ISSUE TITLE
Replicating the target domain training results

### ISSUE THREAD

**Issue by dangnguyenngochai** [ASSERT-KTH/VRepair#10]
Hello @chenzimin,
Can you help me take a look at this log? Somehow my results showed better results than yours using a smaller architecture with half the size?
Here is my log: [log file](https://raw.githubusercontent.com/dangnguyenngochai/VRepair2.0/main/param_sweep_tgt/0_parameter_sweep/log.txt?token=GHSAT0AAAAAABTVPOIGD4RDVDBHM4SPRZTCYT3U53A)

**Comment by chenzimin** [comment]
Hi @dangnguyenngochai,

I could not open the log file.

**Comment by dangnguyenngochai** [comment]
> Hi @dangnguyenngochai,
> 
> I could not open the log file.

https://raw.githubusercontent.com/dangnguyenngochai/VRepair2.0/main/param_sweep_tgt/0_parameter_sweep/log.txt?token=GHSAT0AAAAAABTVPOIGD4RDVDBHM4SPRZTCYT3U53A

Here the url. Does this work ?

**Comment by dangnguyenngochai** [comment]
s

**Comment by chenzimin** [comment]
I can see the log with the new link. The training accuracy is really high (almost 100%), but the validation accuracy is only `[2022-05-06 01:14:23,334 INFO] Validation accuracy: 49.948`. So it seems like it is overfitting to the training data.

**Comment by dangnguyenngochai** [comment]
This log is the training log for target domain only, isn't it too good to
be true comparing to the results you reported in your paper ?

**Comment by chenzimin** [comment]
I believe that OpenNMT-py reports per token accuracy, whereas we calculate the sequence accuracy.

**Comment by dangnguyenngochai** [comment]
Do you have the code for calculating sequence level accuracy cause I can
not seem to find it in the repo or do I have to reimplement the code myself?

**Comment by SteveKommrusch** [comment]
We use the src/compare.py file to compare the output of our model when given test samples with the expected sequence for the sample. For example: "python ../../src/compare.py --src=xlate.txt --tgt=tgt-test.txt -v > pass.txt".

The usage for compare.py is: 
 Usage: python compare.py --src [src_file] --tgt [tgt_file] [-v]
     the -v option will print out all passing cases
   Example: python compare.py --src pred-test.txt --tgt tgt-test.txt

**Comment by chenzimin** [comment]
One more thing, you can use [src/find_best_model_and_translate_config.py](https://github.com/SteveKommrusch/VRepair/blob/main/src/find_best_model_and_translate_config.py) to generate command to generate the predictions from the model with the best validation accuracy.
